lib/tables.c

```diff
@@ -210,7 +210,7 @@ node_table_copy(node_table_t *self, node_table_t *dest)
 
 int WARN_UNUSED
 node_table_set_columns(node_table_t *self, size_t num_rows, uint32_t *flags, double *time,
-        population_id_t *population, char *metadata, uint32_t *metadata_offset)
+        population_id_t *population, const char *metadata, uint32_t *metadata_offset)
```

Comment:
Good cleanup... definitely need to sort out the const'iness of these signatures before finalising. 'const' is my least favourite C feature... definitely causes more problems than it solves for me.